app/screens/home/bottomTabs/ChildDevelopment.tsx

```diff
@@ -148,16 +149,14 @@ const ChildDevelopment = ({ route, navigation }: any): any => {
     filteredData = { ...filteredData, name: item.name };
     setSelectedChildDevData(filteredData);
     const childData = await userRealmCommon.getFilteredData<ChildEntity>(ChildEntitySchema, 'uuid == "' + activeChild.uuid + '"');
-    let milestonefilteredData = await MileStonesData.filter((x: any) => x.child_age.includes(item.id));
-    milestonefilteredData = milestonefilteredData.map((item: any) => ({ ...item, toggleCheck: false }))
+    const milestonefilteredData = await MileStonesData.filter((x: any) => x.child_age.includes(item.id));
```

Comment:
Let's please rename it to  filteredMilestoneData.

Reply:
okay sure I will do that